fe/fe-core/src/main/java/com/starrocks/service/FrontendOptions.java

```diff
@@ -129,15 +129,22 @@ private static void analyzePriorityCidrs() {
 
         String[] cidrList = priorCidrs.split(PRIORITY_CIDR_SEPARATOR);
         List<String> priorNetworks = Lists.newArrayList(cidrList);
-        for (String cidrStr : priorNetworks) {
-            priorityCidrs.add(new CIDR(cidrStr));
-        }
+        priorityCidrs.addAll(priorNetworks);
     }
 
     private static boolean isInPriorNetwork(String ip) {
-        for (CIDR cidr : priorityCidrs) {
-            if (cidr.contains(ip)) {
-                return true;
+        for (String cidr : priorityCidrs) {
+            cidr = cidr.trim();
+            if (!cidr.contains("/")) {
+                // it is not valid CIDR, compare ip directly.
+                if (cidr.equals(ip.trim())) {
```

Comment:
should be case insensitive or not ?

Reply:
this input only have number and not. it does not need to consider case insensitive.